src/persist-types/src/codec_impls.rs

```diff
@@ -94,3 +94,9 @@ impl Codec64 for u64 {
         u64::from_le_bytes(buf)
     }
 }
+
+impl Opaque for u64 {
```

Comment:
`i64`? Not sure it matters if it's getting changed to the `Epoch` struct though.

Reply:
yeah I was thinking I didn't want to introduce the`i64` impl and then forget to remove it once we have `Epoch`. I don't feel strongly about that though. u64 is just the one used in persist tests (which maybe want to eventually use their own `Epoch` equivalant, but I was being lazy)